frontend/packages/integration-tests-cypress/views/common.ts

```diff
@@ -1,2 +1,11 @@
+import { expect } from 'chai';
```

Comment:
Do we need this import?  I've been using `expect` without declaring an import.

Reply:
I was getting this weird Typescript error without it: https://stackoverflow.com/questions/40406728/typescript-throw-error-property-to-does-not-exist-on-type-matchers-when-comp

If you have suggestions, I'm here for them!